stat_var_renaming/stat_var_renaming.py

```diff
@@ -0,0 +1,631 @@
+
+  Args:
+    stat_vars -> The dataframe of statistical variable with 
+      HumanReadableNames generated.
+    client -> An authenticated BigQuery client.
+  """
+  stat_vars_query = """
+  SELECT distinct id FROM
+  `google.com:datcom-store-dev.dc_kg_2020_07_12_01_01_43.Instance` WHERE type =
+  "StatisticalVariable"
+  """
+  existing_stat_vars = client.query(stat_vars_query).to_dataframe()['id']
+  return stat_vars[stat_vars['HumanReadableName'].isin(existing_stat_vars)]
+
+def create_human_readable_names(stat_vars, client):
+  """ Handles generating human readable statistical variables from BQ response.
+  This function handles renaming constraints, populations, and values; 
+  removing dependent constraints; and adding a column for the HumanReadableName.
+
+  Args:
+    stat_vars -> Raw dataframe of statistical variables as returned by the
+      download_stat_vars function.
+    client -> Authenticated BigQuery dataframe.
+
+  Returns:
+    stat_vars dataframe with new HumanReadableName column.
+  """
+  # Build constraint remappings.
+  prop_remap={}
+  rename_naics_codes(prop_remap, client)
+  rename_dea_drugs(prop_remap, client)
+  rename_isic_codes(prop_remap, client)
+  cause_of_death_remap(prop_remap, client)
+  prefix_strip(prop_remap)
+  rename_boolean_variables(prop_remap, stat_vars)
+  remap_numerical_quantities(prop_remap)
+  pre_postpend_text(prop_remap)
+  misc_mappings(prop_remap)
+
+  # Drop erroneous constraints.
+  for c in range(1, MAX_CONSTRAINTS + 1):
+    stat_vars = stat_vars.query(f"v{c} != 'NAICSUnknown'")
+
+  # Apply constraint renamings.
+  stat_vars=stat_vars.apply(
+      lambda row: remap_constraint_from_prop(row, prop_remap), axis=1)
+      
+  # Remove dependent constraints.
+  stat_vars = remove_depedent_constraints(stat_vars)
+  stat_vars = left_fill_columns(stat_vars)
+
+  # Generate human readable names.
+  stat_vars['HumanReadableName'] = stat_vars.apply(row_to_human_readable)
+
+  # Manually rename special case.
+  stat_vars.loc[stat_vars['HumanReadableName'] == 'Count_Death_Medicare',
+      'HumanReadableName'] = "Count_Death_MedicareEnrollee"
+  return stat_vars
+
+def output_stat_var_documentation(stat_vars):
+  """ Outputs markdown file of Statistical Variable list for documentation.
+
+  Outputs Statistical Variable list as a dropdown menu organized by verticals
+  and population type. Some verticals have no sub-population type grouping.
+  These groupings are defined in STAT_VAR_POPULATION_GROUPINGS. The markdown
+  file is output to statistical_variables.md.
+
+  Args:
+    stat_vars -> The dataframe of statistical variable with 
+      HumanReadableNames generated.
+  """
+  # Dynamically generate list of disasters.
+  natural_disasters = []
+  for popType in stat_vars['populationType'].unique():
+    if "Event" in popType:
+      natural_disasters.append(popType)
+  # (False, True) -> Only group disaster statistical variables by populationType
```

Comment:
Can you make this a top-level warning in the README?

Reply:
Added